rero_ils/modules/acq_orders/models.py

```diff
@@ -40,3 +40,13 @@ class AcqOrderMetadata(db.Model, RecordMetadataBase):
     """AcqOrder record metadata."""
 
     __tablename__ = 'acq_order_metadata'
+
+
+class AcqOrderStatus:
+    """Available statuses about an Acquisition Order."""
+
+    CANCELED = 'canceled'
+    ORDERED = 'ordered'
+    PENDING = 'pending'
```

Comment:
PENDING has been removed no ? 
It's "approved" if I'm right.

Reply:
APPROVED for order_line ; PENDING for order